### PR TITLE
Messaging export 57

### PR DESCRIPTION
**Pull request by pmadria**
This is built on Mick's current pull request with the button to generate the questionnaire response after clicking the button. Another button has been created and it exports the information via a console.log at the moment.

### CODE REVIEW

**Comment by mickohanlon23** [comment]
Do you want to pull in the recent changes from master? It looks like you branched off of a slightly outdated version of my questionnaireResponse button functionality, which was just merged.

**Comment by pmadria** [comment]
changes are now on top of mick's changes

File: src/components/Abstractor/Abstractor.tsx [hunk 1]
```diff
@@ -58,11 +58,17 @@ const Abstractor = ({ patientData, library, valueSetMap, questionnaire }: Props)
...
+    const exporter = window.LForms.Util.getFormHL7Data();
+    console.log(exporter);
+  }
+
   return (
     <div>
       <div id="formContainer"> </div>
       <button onClick ={() => generateQR()}>Generate Questionnaire Response</button> 
       {responseGenerated.isGenerated && <p>Questionnaire Response has been generated with {responseGenerated.count} answer(s) and has been logged to the console!</p>}
+      <button onClick ={() => generateHL7Message()}>Get HL7 Information</button>
```
Comment (mickohanlon23) [inline src/components/Abstractor/Abstractor.tsx:71]: I think "Get HL7 Information" is a little vague. I would name it something like "Generate HL7 v2 Message". (Check out one of the forms on NLM form widget to see how they label their v2 message exporter button: https://lhcforms.nlm.nih.gov/lhcforms)

**Review by mickohanlon23 — CHANGES_REQUESTED** [review]
Looks good, the v2 seems to update accordingly as you delete the answer options from the form, showing that it is properly accessing the updated form. I would just mention "v2" on the name of the button.

File: src/components/Abstractor/Abstractor.tsx (hunk 1 above)
Comment (pmadria) [reply src/components/Abstractor/Abstractor.tsx]: yep changed!

**Review by mickohanlon23 — APPROVED** [review]
Looks good. v2 output matches output from NLM widget.

**Review by mgramigna — APPROVED** [review]

**Pull request merged**